tests/test_examples.py

```diff
@@ -1380,3 +1378,26 @@ def test_regressions(tmp_path, request):
             check_subprocess=True,
             uninstall=True,
         )
+
+
+@pytest.mark.xfail(
+    condition=(
+        CONDA_EXE == StandaloneExe.CONDA
+        and check_version(CONDA_EXE_VERSION, min_version="25.5.0", max_version="25.7.0")
+    ),
+    reason="conda-standalone 25.5.x fails with protected base environments and older versions are ignored",
```

Comment:
```suggestion
    reason="conda-standalone 25.5.x fails with protected base environments",
```

I don't think that last aside is necessary since the tests will run and pass for older versions.